addons/stock/models/stock_move.py

```diff
@@ -262,7 +262,7 @@ def _compute_is_quantity_done_editable(self):
         for move in self:
             move.is_quantity_done_editable = move.product_id
 
-    @api.depends('picking_id', 'name')
+    @api.depends('picking_id', 'name', 'picking_type_id')
```

Comment:
doesn't the picking id cause the "move.picking_id.name" to change and this is the source of the change? if it's the case, maybe we should rather use `picking_id.name` as new depends instead?